proj/src/main/webapp/chat.js

```diff
@@ -924,6 +924,41 @@ function addUserInfoToDom(userObj) {
     }
 }
 
+function initDeleteAccountButton(){
+    const txtEmail = document.getElementById("email");
+    const txtPassword = document.getElementById("pass");
+    const btnDelete = document.getElementById("btnDeleteAccount");
+    
+    
+
+    btnDelete.addEventListener("click", async function () {
+        const emailVal = txtEmail.value;
+        const passVal = txtPassword.value;
+
+        // Re-authenticate credentials
+        const auth = firebase.auth();
+        const promise = auth.signInWithEmailAndPassword(emailVal, passVal).then(function(){
+            // Unsubscribe from chats 
+            notifications.unSubscribeFromAllChats;
```

Comment:
I think you're missing the parentheses at the end to actually call the method.

Reply:
oh gotcha, thanks